### ISSUE TITLE
Typo in test setup of step_create_task_test

### ISSUE THREAD

**Issue by hferentschik** [jenkins-x/jx#4052]
There is a typo in specifying the _testVersionsDir_:

```go
testVersionsDir := path.Join("test_data", "cmmon_versions")
```

Should be 'common_versions'. If changed, tests will fail, since now the dir will be picked up and some versions in the checked in YAML files won't match. 

Need to update all test files.

**Comment by abayer** [comment]
Ha, that's sad. I'll work on it as soon as I can. =)

**Comment by abayer** [comment]
Fixed this in #4064 but forgot to link.
